File: python/calculator.py
def calculate(x, operator, z):
    if operator == "+":
        return x + z
    elif operator == "-":
        return x - z
    elif operator == "*":
        return x * z
    elif operator == "/":
        return x / z
    else:
        return "Invalid operator"

def main():
    expression = input("Enter expression (e.g., 2 + 2): ")
    x, operator, z = expression.split()
    x = int(x)
    z = int(z)
    result = calculate(x, operator, z)
    print(f"Result: {result}")

if __name__ == "__main__":
    main()

File: js/calculator.js
function calculate(x, operator, z) {
    switch (operator) {
        case '+':
            return x + z;
        case '-':
            return x - z;
        case '*':
            return x * z;
        case '/':
            return x / z;
        default:
            return 'Invalid operator';
    }
}
